Automattic/wp-calypso
# Subscription Manager: Create TabsSwitcher component and add tabs on Subscription Management page

Resolves #74075

Pairing with @mashikag 

## Proposed Changes

* Create TabsSwitcher component
* Create SubscriptionManagementPage component
* Add Sites, Comments, and Settings tabs on SubscriptionManagementPage
* Redirect the base route to the default tab (/sites)

<img width="636" alt="Screenshot 2023-03-14 at 18 21 23" src="https://user-images.githubusercontent.com/3113712/225148961-04cbe27f-5313-41fe-8fa3-22eab7fa9697.png">

## Testing Instructions

1. Pull this branch to your local env
2. Start your calypso
3. Open http://calypso.localhost:3000/subscriptions
4. Verify if it redirects to the default tab (http://calypso.localhost:3000/subscriptions/sites)
5. Verify if the tabs are working as expected (changing content and browser location)

## Pre-merge Checklist
